### ISSUE TITLE
[Feature request] Hide assets folder inside collection

### ISSUE THREAD

**Issue by pove** [usebruno/bruno#787]
When you add an assets folder to store files that you will use in scripts, this assets folder will be available in Bruno collections menu. There are several possibilities, one could be always hide assets folder name, other could be to have a new option in bruno.json file to add folders we want to hide.

**Comment by helloanoop** [comment]
> other could be to have a new option in bruno.json

I like this approach. I think we can call the key `ignore` (inspired by `.gitignore`, `.dockerignore`)

```json
{
  "version": "1",
  "name": "bruno-testbench",
  "type": "collection",
  "ignore": [
    "assets",
    "logs"
  ]
}
```

**Comment by martinsefcik** [comment]
@helloanoop Probably this approach won't be needed in case collection folders will have own bru file (with folder presets - scripts, auth, headers, ...). Then each folder with bru file will be displayed in collection and others will be ignored.

**Comment by helloanoop** [comment]
@martinsefcik

Consider the case when the user creates an empty folder (with intent to add requests under it). We have to display the empty folder right?

**Comment by martinsefcik** [comment]
@helloanoop Do you mean if he creates empty folder outside bruno-app? If yes, then OK. But If you mean inside bruno-app then I expect that with creating empty folder also this folder's bru file will be created automatically in the future. So in such case it wouldn't be a problem, folder will be displayed in gui even there is no request because there will be this folder's bru file.

But your approach is also OK for me.

**Comment by Andreas-Schoenefeldt** [comment]
Encountering this issue just now also, I would prefer the `ignore` property in the bruno.json

Thx for making this great tool!

**Comment by romain030992** [comment]
Duplicate of #1496 which has been resolved?

**Comment by helloanoop** [comment]
This is now available in `v1.8.0`

You can add the directories to be hidden in `ignore` inside your `bruno.json`
```
{
  "version": "1",
  "name": "bruno-testbench",
  "type": "collection",
  "ignore": [
    "assets",
    "logs"
  ]
}
```

**Comment by keawade** [comment]
> This is now available in `v1.8.0`
> 
> You can add the directories to be hidden in `ignore` inside your `bruno.json`
> 
> ```
> {
>   "version": "1",
>   "name": "bruno-testbench",
>   "type": "collection",
>   "ignore": [
>     "assets",
>     "logs"
>   ]
> }
> ```

Can this still be done in v3+ in the `opencollection.yml` file? I'm not finding documentation of it anywhere and this formation of the config isn't working.
